nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/RunWorker.cs

```diff
@@ -229,7 +229,7 @@ async Task TrackOriginalContentsAsync(string directory, string fileName, string?
                         // TODO: need to report if anything was actually updated
                         if (updateResult.ErrorType is null || updateResult.ErrorType == ErrorType.None)
                         {
-                            if (dependencyLocation != dependencyFilePath)
+                            if (dependencyLocation != dependencyFilePath.EnsurePrefix("/"))
```

Comment:
Is this part still needed?  The call to `.FullyNormalizedRootedPath()` a few lines up should have handled it.

Reply:
I remember seeing that the prefix was missing in some cases at one point. But you're right that's no longer the case. I removed it.